indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -103,35 +108,96 @@ function SidebarMenu({
-                 vertical
-                 width="thin"
-                 direction="right"
-                 onHide={onClickOption}
-                 inverted
-                 visible={visible}
-                 styleName="sidebar">
-            {options.map(({key, text, icon, onClick, iconColor, active, tooltip}) => {
-                const item = (
-                    <Menu.Item as="a" key={key} active={active} onClick={() => {
-                        onClick();
-                        if (onClickOption) {
-                            onClickOption();
-                        }
-                    }}>
-                        <Icon name={icon} color={iconColor} />
-                        {text}
-                    </Menu.Item>
-                );
-                if (!tooltip) {
-                    return item;
-                }
-                return <Popup trigger={item} content={tooltip} key={key} position="left center" />;
-            })}
-        </Sidebar>
+        <>
+            <Sidebar as={Menu}
+                     animation="overlay"
+                     icon="labeled"
+                     vertical
+                     width="thin"
+                     direction="right"
+                     onHide={onClickOption}
+                     inverted
+                     visible={visible}
+                     styleName="sidebar">
+                {options.map(({key, text, icon, onClick, iconColor, active, tooltip}) => {
+                    const item = (
+                        <Menu.Item as="a" key={key} active={active} onClick={() => {
+                            onClick();
+                            if (onClickOption) {
+                                onClickOption();
+                            }
+                        }}>
+                            <Icon name={icon} color={iconColor} />
+                            {text}
+                        </Menu.Item>
+                    );
+                    if (!tooltip) {
+                        return item;
+                    }
+                    return <Popup trigger={item} content={tooltip} key={key} position="left center" />;
+                })}
+                <div styleName="bottom-align">
+                    {helpURL && (
+                        <Menu.Item as="a" onClick={() => {
+                            window.location.href = helpURL;
+                        }}>
+                            <Translate>Help</Translate>
+                        </Menu.Item>
+                    )}
+                    {contactEmail && (
+                        <Menu.Item onClick={() => setContactVisible(true)}>
+                            <Translate>Contact</Translate>
+                        </Menu.Item>
+                    )}
+                    {(tosURL || tosHtml) && (
+                        <Menu.Item onClick={() => {
+                            if (tosURL) {
+                                window.location.href = tosURL;
```

Comment:
same as above

Reply:
FWIW, if a text is used you can still set the href to the existing tos page (in the indico core) and preventDefault the click event to avoid going there when opening the modal